test/util/fixtures/ClientServiceFixtures.java

```diff
@@ -0,0 +1,57 @@
+package util.fixtures;
+
+import entities.Client;
+import enums.Roles;
+import enums.StatusEnum;
+
+public final class ClientServiceFixtures {
+
+  private Client client1 = new Client();
```

Comment:
Why don't you creat one single `buildClient` where you have a default one and then you send a client with the desired props to set. No need to repeat the same code over and over again

Reply:
solve